blackduck-artifactory-common/src/main/java/com/synopsys/integration/blackduck/artifactory/modules/inspection/model/PolicyStatusReport.java

```diff
@@ -41,11 +41,11 @@
     public static PolicyStatusReport fromVersionBomComponentView(VersionBomComponentView versionBomComponentView, BlackDuckService blackDuckService) throws IntegrationException {
         PolicySummaryStatusType policySummaryStatusType = versionBomComponentView.getPolicyStatus();
 
-        List<VersionBomPolicyRuleView> versionBomPolicyRuleViews = blackDuckService.getResponses(versionBomComponentView, VersionBomComponentView.POLICY_RULES_LINK_RESPONSE, true);
+        List<VersionBomPolicyRuleView> versionBomPolicyRuleViews = blackDuckService.getAllResponses(versionBomComponentView, VersionBomComponentView.POLICY_RULES_LINK_RESPONSE);
         List<PolicySeverityType> policySeverityTypes = versionBomPolicyRuleViews.stream()
-                                                                 .map(VersionBomPolicyRuleView::getSeverity)
-                                                                 .map(severity -> EnumUtils.getEnumIgnoreCase(PolicySeverityType.class, severity))
-                                                                 .collect(Collectors.toList());
+                                                           .map(VersionBomPolicyRuleView::getSeverity)
+                                                           .map(severity -> EnumUtils.getEnumIgnoreCase(PolicySeverityType.class, severity))
+                                                           .collect(Collectors.toList());
 
```

Comment:
I feel like you can use a better api of Black Duck Service here - the one that takes a Predicate.